src/config/Pf4Icon.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
```

Comment:
Would it make sense to also name the file KialiIcon.tsx?

Reply:
Oh right. I forgot about changing that when I renamed.